packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1278,7 +1278,7 @@ export default class ScheduledPaymentModule {
       // ExceedMaxGasPrice: gasPrice must be lower than or equal maxGasPrice
       // PaymentExecutionFailed: safe balance is not enough to make payments and pay fees
       // OutOfGas: executionGas to low to execute scheduled payment
-      throw extractSendTransactionError(e, new Interface(ScheduledPaymentABI));
+      throw new Error(extractSendTransactionError(e, new Interface(ScheduledPaymentABI)));
```

Comment:
Because we were throwing a string here, this piece of code in the executor was trying to call `message` on a string, which resulted in a blank value:

https://github.com/cardstack/cardstack/blob/b19a1f838a2d95de25cdf43324a5de431c6a2d65/packages/hub/services/scheduled-payments/executor.ts#L160